packages/react-shared-contexts/src/ProviderContext/ProviderContext.ts

```diff
@@ -3,7 +3,7 @@ import { ProviderContextValue } from './ProviderContext.types';
 
 export const ProviderContext = React.createContext<ProviderContextValue>({
   targetDocument: typeof document === 'object' ? document : undefined,
-  dir: 'ltr',
+  dir: document.dir as 'ltr' | 'rtl',
```

Comment:
This will immediately throw in SSR:

![image](https://user-images.githubusercontent.com/14183168/129410534-6a4fb809-2810-424e-80c9-a86cade0a8f0.png)

https://codesandbox.io/s/dank-wildflower-w6smd?file=/pages/index.js

----

If we will have this conditionally this will be also unsafe for SSR: values will be different on server (no DOM) and client (where `document.dir` might be RTL). React Spectrum handles it in more complicated way: https://github.com/adobe/react-spectrum/blob/f4d4fe23b08bc0073259dea3d16f3c652e67ee57/packages/%40react-aria/i18n/src/useDefaultLocale.ts#L69-L76